### ISSUE TITLE
manually pull and push required

### ISSUE THREAD

**Issue by aisbergde** [0xGG/crossnote-app#135]
the git repository used by crossnote is a local clone and there is no external access for other apps. If I want to work with the repository in alternative apps I need to be able to manually pull and push changes. I can use another git client to create a clone in a folder with access by other apps. Currently this is not possible, there is only some automatic check which is set by default to 60 minutes.

I also don't understand how to use crossnote on different devices if I can't manually push and pull changes.

**Comment by shd101wyy** [comment]
![Screenshot from 2020-06-29 10-41-51](https://user-images.githubusercontent.com/1908863/85967566-57d62700-b9f5-11ea-9e48-f28407711bac.png)

These two buttons are for push and pull

**Comment by aisbergde** [comment]
sorry, I didn't see. I was looking for a way to push and pull on the repository level. And when using on android the buttons are only visible after scrolling and it didn't come in my mind to search this function on notes level. But now I know.

**Comment by shd101wyy** [comment]
Yeah the UI design has some problem. I am going to improve it in the future. Thanks for your feedback

**Comment by dongshunyao** [comment]
The project is being refactored, so we will close outdated issues. Once the new version is released, please open a new issue if you encounter any problems. Thank you for your contributions and support!
